elliptic.js

```diff
@@ -1 +1,42 @@
-module.exports = require('./lib')(require('./lib/elliptic'))
+let BigIntSupportChecked = false;
+let secp256k1;
+
+if (!BigIntSupportChecked)
+  try {
+    BigInt(1);
+    const noble_secp256k1 = require("@bitcoinerlab/secp256k1");
+    secp256k1 = {
```

Comment:
@alexandrius how do we make sure tests test all implementations? maybe we should move this to a separate file instead of hijacking elliptic.js?

Reply:
@mvayngrib Accidentally deleted description. 
Adding it back:
> TODO: Refactor elliptic instead

Reply:
isn't elliptic still useful when BigInt is not available? And why refactor elliptic if elliptic refers to the elliptic library

Reply:
i was thinking we'd have this here https://github.com/ExodusMovement/secp256k1-node/blob/6cfdf24f1f4adc3cd3a10f721e71d979e14ae215/index.js#L1-L5
```js
try {
  module.exports = require('./bindings')
} catch (err) {
  try {
    module.exports = require('./bitcoinerlab')
  } catch (err) {
    module.exports = require('./elliptic')
  }
}
```

and then we'd make sure all of those 3 implementations pass the test suite

Reply:
> isn't elliptic still useful when BigInt is not available? And why refactor elliptic if elliptic refers to the elliptic library

The file in this repo `elliptic.js`

> isn't elliptic still useful when BigInt is not available? 

Do we need to support non bigint version tho?

This PR is by no means final version. Wanted to leave my work somewhere

Reply:
> Do we need to support non bigint version tho?

maybe not. But let's kill rm elliptic completely then. I just don't want to be confused about which implementation is which

> This PR is by no means final version. Wanted to leave my work somewhere

of course! my comments are probably premature 👍

Reply:
@mvayngrib Any point in preserving `bindings.js` considering we have native build removed?
https://github.com/ExodusMovement/secp256k1-node/commit/95b716d43858c70f8bc7c62d8c67736c07b39df6#diff-5bb8db779819ddef5956a5d9d5949c05ef7445237656ca37bf2f02720271440b

Reply:
good find, let's rm them